backend/director/core/session.py

```
@@ -220,28 +220,31 @@ class OutputMessage(BaseMessage):
     def update_status(self, status: MsgStatus):
-        """Update the status of the message and publish the message to the socket. for loading state."""
+        """Update the status and broadcast to session room."""
         self.status = status
         self._publish()
 
     def push_update(self):
-        """Publish the message to the socket."""
+        """Push real-time update to session room."""
         try:
             self._publish()
         except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)